documentation/infrastructure_crl.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: "Puppet Server: Intermediate CA"
+canonical: "/puppetserver/latest/infrastructure_crl.html"
+---
```

Comment:
Can we get this page added to the TOC?

Reply:
Yes

Reply:
I think the sidebar nav is controlled by _puppetserver_nav.html in this directory?

Reply:
Okay @MFredette wasn't sure. I can update that then.

Reply:
Grr... Sorry, it looks like I'm wrong... I'm not sure what this is used for (maybe it was used before they switched doc sites? maybe we should delete it) From looking at the content on the live site it looks like the index file in this directory includes a TOC on Puppet Server's index page, while the left hand nav is controlled by [puppet-docs](https://github.com/puppetlabs/puppet-docs/blob/master/source/puppet/6.0/_puppet_toc.html#L105). This is both side-by-side https://puppet.com/docs/puppetserver/6.0/index.html

Reply:
Well I think updating the index is good, so I'm gonna leave this. I would also kind of like to update the left nav, I feel like it could be organized more clearly.